Tests/SonarQube.Bootstrapper.Tests/BootstrapperExeTests.cs

```diff
@@ -164,41 +191,6 @@ public void Exe_PreProc_VersionCheckSucceeds_PreProcSucceeds()
         }
 
         [TestMethod]
-        [Description("If the host url is not specified, a default is used")]
-        public void Exe_PreProc_DefaultHostUrl()
-        {
```

Comment:
No longer applicable: the pre-processor no longer has a hard-coded default it will use if one isn't specified on the command line / in a settings file.